src/main/resources/templates/layout.html

```diff
@@ -14,7 +14,7 @@
 	<link rel="stylesheet" href="//use.fontawesome.com/releases/v5.7.2/css/all.css">
 
 	<!-- Email Encryption: https://jumk.de/nospam/stopspam.html -->
-	<script type="text/javascript"> <!--
```

Comment:
Der Kommentar, den du hier gelöscht hast, wird immernoch in Zeile 34 geschlossen. Schau dir das vielleicht nochmal an